happy-travel/edo
# Feature/aa 1551 add multicurrency to search

Issue: https://github.com/happy-travel/agent-app-project/issues/1551
Insomnia: Edo -> Agent -> Availability -> 3 search steps
Not tested yet

Test cases: 
1) Ensure that currency for wide availability search (first step) was taken from agent's preferred currency
2) Ensure that currency for room selection search (second step) was taken from agent's preferred currency
3) Ensure that currency for booking evaluation search (third step) was taken from agent's preferred currency
